demo-app/package.json

```diff
@@ -5,8 +5,10 @@
   "dependencies": {
     "font-awesome": "^4.7.0",
     "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "react-scripts-ts": "^3.1.0"
+    "react-dom": "16.8.*",
+    "react-scripts-ts": "^3.1.0",
+    "react-redux": "6.0.1",
+    "redux": "4.0.1"
```

Comment:
Not sure if it's a good idea to put this here as a dependency, mostly because of PatternFly. I'd rather see it as a `devDependency` and inject the used parts from the application where the tree is being used. Not sure if it's doable, more like a philosophical question than me being smart here ...

Reply:
Nice catch. This is really tricky because you should not ship component mounted to redux, imagine if someone were to use 2 trees at the same page with same store. That would be impossible. If you really want to ship some redux things you can ship actions, action types and reducers. But not connected components.

Reply:
What do you mean under "inject"? Can I ship the ConnectedNode or the ConnectedTree for example?